### PR TITLE
Added Ben Simpson as participant

### PR DESCRIPTION
**Pull request by bsimpson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Kim Crayton (Host, [Twitter](https://twitter.com/KimCrayton1), [GitHub](https://github.com/kmcrayton7))
 - Gregor Martynus (Host, [Twitter](https://twitter.com/gr2m), [GitHub](https://github.com/gr2m))
 - Dean Friedland
+- Ben Simpson (Mentor)
```
Comment (hzoo) [inline README.md:16]: Do you want to put a twitter/github?

Comment (gr2m) [inline README.md:16]: yeah sorry, if you want to add it you can send another PR :)
